apps/experiments/models.py

```diff
@@ -345,8 +344,7 @@ def archive(self):
         self.experiments_pre.update(pre_survey=None, audit_action=AuditAction.AUDIT)
         self.experiments_post.update(post_survey=None, audit_action=AuditAction.AUDIT)
 
-    @property
-    def version_details(self) -> VersionDetails:
+    def _get_version_details(self) -> VersionDetails:
```

Comment:
Why not make this a `cached_property`?

Reply:
Just so that we can centralise the caching